package.json

```diff
@@ -118,22 +118,22 @@
     "gulp-rename": "^1.4.0",
     "gulp-svgmin": "^1.2.3",
     "gulp-theo": "^2.0.0",
-    "husky": "^1.1.2",
+    "husky": "^1.2.0",
```

Comment:
I tested this by: 
- clearing `node_modules`
- clearing git hooks
- npm install
- add eslint error to javascript file
- observe husky doing it's thing